### PR TITLE
Added suggested fields.

### PR DESCRIPTION
**Pull request by mooreds**
Expanded more on suggested fields for connector setup.

### CODE REVIEW

File: site/docs/v1/tech/connectors/index.adoc [hunk 1]
```diff
@@ -51,6 +51,13 @@ If you are using the generic connector, you are building the `user` object, so e
 
 In addition, ensure that the same `user.id` is returned each time a user is retrieved. If you have an LDAP attribute in the form of a UUID or a value that can be translated to a UUID, using that attribute or its translation is recommended. If you have to generate a UUID, ensure that you are able to return the same value each time the user authenticates. Doing so ensures that FusionAuth knows which user is being referenced, even if the username or email address changes.
 
+==== Suggested fields
+
+If you are authenticating a user, you likely want to populate these fields:
+
+* `user.registrations`: providing at least one entry in this array will associate a user with an application within FusionAuth, authorizing them to access the application.
+* `user.tenantId`: the tenant that this user should be associated with. Most useful when you have multiple tenants.
```
Comment (robotdan) [inline site/docs/v1/tech/connectors/index.adoc:59]: Wouldn't this be dictated by the tenant that you're authenticating against?

File: site/docs/v1/tech/connectors/index.adoc [hunk 2]
```diff
@@ -68,6 +75,15 @@ If you are using the generic connector, you are building the `user` object, so e
...
 If you don't have a UUID to associate with this user, you may create a random one, as the source datastore won't be consulted after the user is initially migrated. 
 
+==== Suggested fields
+
+If you are migrating a user, you likely want to populate these fields:
+
+* `user.registrations`: providing at least one entry in this array will associate a user with an application within FusionAuth, authorizing them to access the application.
+* `user.data`: allows you to store arbitrary data associated with the user. This can be application or migration specific. For example, you could indicate the migration date of a user for subsequent searches.
+* `user.insertInstant`: the timestamp the user was created, useful for retaining history.
+* `user.tenantId`: the tenant that this user should be associated with. Most useful when you have multiple tenants.
```
Comment (robotdan) [inline site/docs/v1/tech/connectors/index.adoc:85]: Same comment as above.

File: site/docs/v1/tech/connectors/index.adoc (hunk 1 above)
Comment (mooreds) [reply site/docs/v1/tech/connectors/index.adoc]: Good point. I tested and it seems that specifying a different tenant id than the tenant of the application being logged into is associated with doesn't work; the app's tenant wins.

**Comment by mooreds** [comment]
Merging this today @robotdan

**Pull request merged**